palace/models/postoperator.cpp

```diff
@@ -1344,4 +1421,16 @@ PostOperator<ProblemType::DRIVEN>::MeasureDomainFieldEnergyOnly<ProblemType::DRI
 template auto
 PostOperator<ProblemType::DRIVEN>::InitializeParaviewDataCollection(int ex_idx) -> void;
 
+template void ScaleGridFunctions<std::unique_ptr<GridFunction>>(
+    double, int, bool, std::unique_ptr<GridFunction> &, std::unique_ptr<GridFunction> &,
+    std::unique_ptr<GridFunction> &, std::unique_ptr<GridFunction> &);
```

Comment:
This function is templated, but at the end the implementation relies on the fact that the arguments are palace GridFunctions, so maybe we can remove the template?